vscode/src/jdkDownloader.ts

```diff
@@ -220,6 +220,19 @@ export async function extractJDK(jdkTarballPath: string, extractionTarget: strin
   });
 }
 
+const handleJdkPaths = async (directoryName: string, parentPath: string, osType: string): Promise<string> => {
+  let name = directoryName;
+  if (osType === 'macOS') {
+    name = `${directoryName}.jdk`;
+  }
+  const directoryPath = path.join(parentPath, name);
+  if (fs.existsSync(directoryPath)) {
+    await fs.promises.rmdir(directoryPath, { recursive: true });
```

Comment:
I wonder if there should be some warning to (or confirmation from) the user before deleting the directory?

Reply:
Yes it sounds good to have confirmation, I have added it.